tests/unittests/distros/test_user_data_normalize.py

```diff
@@ -160,6 +160,8 @@ def test_users_simple(self):
         self.assertIn("bob", users)
         self.assertEqual({"default": False}, users["joe"])
         self.assertEqual({"default": False}, users["bob"])
+        (users, _groups) = self._norm({"users": []}, distro)
```

Comment:
parenthesis are not necessary here

Reply:
Changed. I was adhering to the style within the test module, but we might as well just fix the style within the module.